gluetool/tests/test_json.py

```diff
@@ -1,6 +1,7 @@
 import json
 import re
 import string
+from __future__ import print_function
```

Comment:
I believe `__future__` imports must be the very first import in a file.

Reply:
right, thanks, resolved